### PR TITLE
added-prisma-orm

### PR DESCRIPTION
**Pull request by aiasha97**
Added Prisma ORM to backend and connected it for postgres.

### CODE REVIEW

**Review by nmosman — DISMISSED** [review]
To me looks good but there’s a conflict with index.ts, can you resolve and then push again? Can help with this too if you’d like

**Comment by aiasha97** [comment]
I made the changes, you can see them in my branch. Do I make a new PR?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules
```
Comment (Aelbannan) [inline .gitignore:1]: Why is there a gitignore in root? Don't think this is needed

**Review by Aelbannan — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (aiasha97) [reply .gitignore:1]: oh I forgot to delete that.

**Pull request merged**